### PR TITLE
Persisted user info into user model upon login

### PR DESCRIPTION
**Pull request by allenwang968**
- [ ] The pull request is complete according to the following criteria:
  - [ ] Acceptance criteria have been met
  - [ ] The documentation is kept up-to-date
  - [ ] Comprehensive tests (if applicable) have been generated and all pass.
  - [ ] The pull request describes the changes that have been made, and enough information is present in the description for any developer to understand what has changed
  - [ ] Commits have been squashed (or will be on merge).
  - [ ] The branch name is descriptive and follows the pull request title format : {issue/bug...}/(Issue Number) - Name of issue. E.g bug/30-Fix-Project
  - [ ] The pull request title is of the following format : {issue/bug...}/(Issue Number) - Name of issue. E.g bug/30-Fix-Project
  - [ ] The description uses [github syntax](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) to link to the issue. E,g Resolves se701g2/Doto#188
  - [ ] At least two reviewers assigned. One of which must be the assigner of the issue.
  - [ ] If there are merge conflicts, run git rebase as opposed to git merge with master.

### CODE REVIEW

**Comment by allenwang968** [comment]
wow good job very nice

**Comment by FocalChord** [comment]
Looks good but can you update the PR description to link the issue you are closing.

**Comment by allenwang968** [comment]
> wow good job very nice

i know

File: doto-backend/index.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const passport = require('passport')
 // Mongoose connection
 const mongoose = require('mongoose');
 // db connection string will point to Azure string only in production, fallbacks to dev database string 
-const connectionString = process.env.AZURE_CONN || process.env.mongodb_uri;
+const connectionString = process.env.AZURE_CONN || process.env.DEVELOPMENT_DB_CONN;
```
Comment (lucycy98) [inline doto-backend/index.js]: think caseys already resolved this but should be fine

File: doto-backend/src/config/passport-setup.js [hunk 2]
```diff
@@ -7,16 +7,19 @@ passport.use(
         callbackURL: '/auth/google/redirect',
         clientID: process.env.GOOGLE_API_CLIENT,
         clientSecret: process.env.GOOGLE_API_SECRET
-    }, (accessToken, refreshToken, profile, email, done) => { 
+    }, (accessToken, refreshToken, profile, done) => { 
+        const email = profile.emails[0].value
```
Comment (lucycy98) [inline doto-backend/src/config/passport-setup.js]: could this throw an error?

**Pull request merged**